database/entity/0002-add_settings/UserSetting.ts

```diff
@@ -1,5 +1,5 @@
 import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, ManyToOne } from 'typeorm'
-import { User } from '../User'
+import { User } from '../0034-drop_server_user_table/User' // Wolle: last fitting one or old was correct?
```

Comment:
Because the new `database/entity/0037-drop_server_user_table/User.ts` file does not fit to this reference I changed it to the last fitting one.

Is that correct @Mogge ? Or shall it reference to the actual for ever?
To leave the reference to a not fitting dataset would be confusing for our future development I think …